packages/dashboard-v2/src/components/LatestActivity/useFormattedActivityData.js

```diff
@@ -10,7 +10,7 @@ const parseFileName = (fileName) => {
 
 const formatItem = ({ size, name: rawFileName, uploadedOn, downloadedOn, ...rest }) => {
   const [name, type] = parseFileName(rawFileName);
-  const date = dayjs(uploadedOn || downloadedOn).format("MM/DD/YYYY; HH:MM");
+  const date = dayjs(uploadedOn || downloadedOn).format("MMM D, YYYY HH:MM");
```

Comment:
Should we move the date format to a central place, so we can change it for the entire site without updating multiple spots?